cypress/components/definition-list/definition-list.cy.tsx

```diff
@@ -44,9 +44,9 @@ context("Testing Definition List component", () => {
     );
 
     it.each([
-      [135, 1229, 10],
-      [683, 683, 50],
-      [1229, 135, 90],
+      [113, 1229, 10],
```

Comment:
**question**: I'm sure you know what you're doing, but it's not obvious to me why the pixel widths of the elements should have changed (by over 20 pixels!) in a fix that surely shouldn't introduce any functional or design changes?

Reply:
Fair point to raise, previously each element was wrapped in div so the width included the 24px of right padding. With these divs removed it isn't including that padding anymore. I've modified the test slightly to set the viewport width to 1000px and then added a constant for the padding that is subtracted in the assertion so hopefully it's a little clearer now what's going on